test/dlc_tests/conftest.py

```diff
@@ -618,6 +606,42 @@ def delete_s3_artifact_copy():
     return conn
 
 
+@pytest.fixture(autouse=True)
+def skip_s3plugin_test(request):
+    if "training" in request.fixturenames:
+        img_uri = request.getfixturevalue("training")
+    elif "pytorch_training" in request.fixturenames:
+        img_uri = request.getfixturevalue("pytorch_training")
+    else:
+        return
+    _, fw_ver = get_framework_and_version_from_tag(img_uri)
+    if request.node.get_closest_marker("skip_s3plugin_test"):
+        if Version(fw_ver) not in SpecifierSet("<=1.12.1,>=1.6.0"):
+            pytest.skip(
+                "s3 plugin is only supported in PT 1.6.0 - 1.12.1, skipping this container with tag{}".format(
+                    fw_ver
+                )
+            )
+
+
+@pytest.fixture(autouse=True)
+def skip_inductor_test(request):    
+    if "training" in request.fixturenames:
+        img_uri = request.getfixturevalue("training")
+    elif "pytorch_training" in request.fixturenames:
+        img_uri = request.getfixturevalue("pytorch_training")
+    else:
+        return
+    _, fw_ver = get_framework_and_version_from_tag(img_uri)
+    if request.node.get_closest_marker("skip_inductor_test"):
```

Comment:
Same question as the `skip_s3plugin_test` fixture.

If the `skip_inductor_test` fixture is meant to be autoused, why do you need to check if it is a marker on the pytest test function? If you need to check that the marker exists, then why not just make this fixture scoped to `session` instead of making it `autouse=True`, so that this function is only run when the marker is applied on a particular test?

Reply:
I noticed that a lot of the other skip_test fixtures have used autouse so I followed this convention. And since we checked the mark in the skip_test, the tests which have not been marked with this skip-test fixture will not be broken